examples/miscellaneous/miscellaneous_ex12/miscellaneous_ex12.C

```diff
@@ -392,7 +392,7 @@ void assemble_shell (EquationSystems & es,
       // First compute element data at the nodes
       std::vector<Point> nodes;
       for (unsigned int i=0; i<elem->n_nodes(); ++i)
-        nodes.push_back(*(elem->node_ptr(i)));
+        nodes.push_back(elem->node_ref(i));
```

Comment:
I think this is a perfect 2-line illustration of the shortcomings of the old interface.